plugin/src/main/java/io/snyk/eclipse/plugin/SnykStartup.java

```diff
@@ -123,7 +109,14 @@ private boolean downloadLS() {
     return true;
   }
 
-  boolean isDownloadAllowed(String customPath) {
+  boolean isDownloadAllowed(Preferences preferences) {
+    return isDownloadAllowed(preferences.getLsBinary(), preferences.isManagedBinaries());
+  }
+  boolean isDownloadAllowed(String customPath, boolean managedBinaries) {
+    if (!managedBinaries) {
+        logger.info("LS: Automatically managed binaries disabled, not downloading");
+        return false;
+      }
     boolean customPathSet = customPath != null && !customPath.isBlank();
     if (customPathSet) {
```

Comment:
This is wrong - this couples managed binaries with custom path - a custom path does not mean that download is not automatic, it just means the automatic download/update should use the custom path as destination path

Reply:
I think this also means that we need changes to LSDownloader, to take the custom path as destination path.

Reply:
Done, that was indeed a hangover from the past. 

I don't think we need to change LSDownloader as it uses the path as defined in preferences.